src/cursor/cur_backup_incr.c

```diff
@@ -206,6 +214,8 @@ __curbackup_incr_next(WT_CURSOR *cursor)
             WT_ERR(WT_NOTFOUND);
         WT_ASSERT(session, cb->granularity != 0);
         WT_ASSERT(session, total_len != 0);
+        __wt_verbose(session, WT_VERB_BACKUP, "Set key with incr block offset %" PRIu64 " gran %" PRIu64,
+          cb->offset + cb->granularity * start_bitoff, total_len);
```

Comment:
Same here, include `cb->incr_file` so we know what file the information is for and include `WT_BACKUP_RANGE`.

Reply:
ta!